python/lsst/rapid/analysis/rubinTv.py

```diff
@@ -70,7 +71,13 @@ def _dataIdToFilename(channel, dataId):
     filename : `str`
         The filename
     """
-    filename = f"{PREFIXES[channel]}_dayObs_{dataId['dayObs']}_seqNum_{dataId['seqNum']}.png"
+    def dayIntToDayObsStr(dayInt):
+        dayStr = str(dayInt)
+        assert len(dayStr) == 8
+        return '-'.join([dayStr[0:4], dayStr[4:6], dayStr[6:8]])
+
+    dayObsStr = dayIntToDayObsStr(dataId['day_obs'])
+    filename = f"{PREFIXES[channel]}_dayObs_{dayObsStr}_seqNum_{dataId['seq_num']}.png"
```

Comment:
Of course butler can do this with `lsst.daf.butler.FileTemplates` but it uses a DatasetRef.